blackboxopt/optimization_loops/dask_distributed.py

```diff
@@ -162,13 +162,13 @@ def run_optimization_loop(
                 logger.info("Optimization is complete")
                 break
 
-        for evaluation in dask_scheduler.check_for_results(timeout_s=20):
-            optimizer.report_evaluation(evaluation)
-            evaluations.append(evaluation)
+        new_evaluation = dask_scheduler.check_for_results(timeout_s=20)
+        optimizer.report(new_evaluation)
+        evaluations.extend(new_evaluation)
 
     while dask_scheduler.has_running_jobs():
-        for evaluation in dask_scheduler.check_for_results(timeout_s=20):
-            optimizer.report_evaluation(evaluation)
-            evaluations.append(evaluation)
+        new_evaluation = dask_scheduler.check_for_results(timeout_s=20)
+        optimizer.report(new_evaluation)
+        evaluations.extend(new_evaluation)
```

Comment:
Seems like this should be plural `new_evaluations` since it can be multiple and is always an iterable IIRC.